corehq/apps/user_importer/helpers.py

```diff
@@ -231,6 +231,8 @@ def update_locations(self, location_codes, domain_info):
                              for code in location_codes]
                 self.logger.add_info(
                     UserChangeMessage.assigned_locations_info(locations))
+            else:
+                self.logger.add_info(UserChangeMessage.assigned_locations_info([]))
```

Comment:
migration for old records to be handled in https://github.com/dimagi/commcare-hq/pull/30287